xml/System.IO/TextWriter.xml

```diff
@@ -4231,17 +4294,18 @@ This member is an explicit interface member implementation. It can be used only
         <param name="buffer">The character array to write data from.</param>
         <param name="index">The character position in the buffer at which to start retrieving data.</param>
         <param name="count">The number of characters to write.</param>
-        <summary>Writes a subarray of characters followed by a line terminator asynchronously to the text string or stream.</summary>
+        <summary>Asynchronously writes a subarray of characters to the text stream, followed by a line terminator.</summary>
         <returns>A task that represents the asynchronous write operation.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- The line terminator is defined by the <xref:System.IO.TextWriter.CoreNewLine> field.  
-  
- The <xref:System.IO.TextWriter> class is an abstract class. Therefore, you do not instantiate it in your code. For an example of using the <xref:System.IO.TextWriter.WriteLineAsync%2A> method, see the <xref:System.IO.StreamWriter.WriteLineAsync%2A?displayProperty=nameWithType> method.  
-  
- ]]></format>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The line terminator is defined by the <xref:System.IO.TextWriter.CoreNewLine> field.
+
+The <xref:System.IO.TextWriter> class is an abstract class. Therefore, you do not instantiate it in your code. For an example of using the <xref:System.IO.TextWriter.WriteLineAsync%2A> method, see the <xref:System.IO.StreamWriter.WriteLineAsync%2A?displayProperty=nameWithType> method.
```

Comment:
This seems like an unusual place for this comment.  I think that it can be deleted, or possibly it could be removed to the remarks of the WriteAsync \<MemberGroup>, if one exists.

Reply:
I see what you mean. Yes, I'll move it to the MemberGroup for WriteLineAsync, in line 3930.
```suggestion
```